src/targets/gpu/kernels/include/migraphx/kernels/math.hpp

```diff
@@ -185,6 +183,15 @@ MIGRAPHX_DEVICE_MATH_FOR(float, abs, ::abs)
 MIGRAPHX_DEVICE_MATH_FOR(double, abs, ::abs)
 MIGRAPHX_DEVICE_MATH_FOR(migraphx::half, abs, ::__habs)
 MIGRAPHX_DEVICE_MATH_FOR(migraphx::bf16, abs, ::fabsf)
+MIGRAPHX_DEVICE_MATH_FOR(float, pow, ::powf)
+MIGRAPHX_DEVICE_MATH_FOR(double, pow, ::pow)
+MIGRAPHX_DEVICE_MATH_FOR(migraphx::half, pow, ::powf)
+MIGRAPHX_DEVICE_MATH_FOR(migraphx::bf16, pow, ::powf)
+MIGRAPHX_DEVICE_MATH_FOR(float, fmod, ::fmodf)
+MIGRAPHX_DEVICE_MATH_FOR(double, fmod, ::fmod)
+MIGRAPHX_DEVICE_MATH_FOR(migraphx::half, fmod, ::fmodf)
+MIGRAPHX_DEVICE_MATH_FOR(migraphx::bf16, fmod, ::fmodf)
+MIGRAPHX_DEVICE_MATH_FOR(migraphx::bf16, mod, ::fmodf)
```

Comment:
These should be using the `MIGRAPHX_DEVICE_MATH_WRAP` macro. This only for the math functions that have a simple generic implemention below. These functions do not.